protected/controllers/PostController.php

```diff
@@ -1,94 +1,36 @@
 <?php
+/**
+ * Post Controller Class
+ */
 class PostController extends ControllerCore
 {
-    public function actionWrite() : void
-    {
-        $user = $this->getModel('user')->getCurrent();
-
-        if (!$user->isBan()) {
-            $this->redirect('/user/ban/');
-        }
-
-        if (!$user->isHuman()) {
-            $this->redirect('/user/check/');
-        }
-
-        list($status, $errors) = $this->getModel('post')->create($this->post);
-
-        if ($status) {
-            $user->setFlashSuccess();
-            $this->redirect($user->_getRedirectURL());
-        }
+    /**
+     * @var string Template Scope
+     */
+    public $templaterScope = 'site';
```

Comment:
Codacy found an issue: [Member variable "$templaterScope" is not in valid snake_case format, try "$templater_scope"](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)